### PR TITLE
UML 75 Added session middleware

### PR DESCRIPTION
**Pull request by nsmithuk**

### CODE REVIEW

File: service-viewer/app/src/App/templates/app/enter-code.phtml [hunk 1]
```diff
@@ -25,11 +25,9 @@
                             <label class="govuk-label" for="share-code">
                                 LPA code
                             </label>
-                            <input class="govuk-input govuk-input--width-10" id="share-code-one" name="share-code-one"
+                            <input class="govuk-input" id="share-code-one" name="share-code"
```
Comment (taylorsj1980) [inline service-viewer/app/src/App/templates/app/enter-code.phtml]: Change id attribute to `share-code` ?

File: service-viewer/app/src/App/src/Handler/HomePageHandler.php [hunk 2]
```diff
@@ -38,65 +36,6 @@ public function __construct(
 
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
-        if ($this->template === null) {
```
Comment (taylorsj1980) [inline service-viewer/app/src/App/src/Handler/HomePageHandler.php]: `$this->containerName` and `$this->router` aren't required anymore. Can we remove them from the constructor and adjust the factory?

Comment (nsmithuk) [reply service-viewer/app/src/App/src/Handler/HomePageHandler.php]: Yes - but I'd expect that to fall under your handlers ticket.

File: service-viewer/app/src/App/templates/app/enter-code.phtml (hunk 1 above)
Comment (nsmithuk) [reply service-viewer/app/src/App/templates/app/enter-code.phtml]: Fixed.

**Pull request merged**